CHANGELOG.md

```diff
@@ -7,6 +7,18 @@
 - Improve touch event component info if annotated with [`@sentry/babel-plugin-component-annotate`](https://www.npmjs.com/package/@sentry/babel-plugin-component-annotate) ([#3899](https://github.com/getsentry/sentry-react-native/pull/3899))
```

Comment:
maybe remove this from the changelog?

Reply:
Thanks good point, I'll update the changelog.